packages/theme-chalk/src/mixins/_var.scss

```diff
@@ -0,0 +1,13 @@
+@mixin set-css-color-primary($i) {
+  --el-color-primary-light-#{$i}: #{map.get($--colors, 'primary', 'light-#{$i}')};
+}
+
+@mixin set-css-color-type($type) {
+  --el-color-#{$type}: #{map.get($--colors, $type, 'base')};
+  --el-color-#{$type}-light: #{map.get($--colors, $type, 'light')};
+  --el-color-#{$type}-lighter: #{map.get($--colors, $type, 'lighter')};
+}
+
+@mixin set-css-var-type($name, $type, $--variables) {
+  --el-#{$name}-#{$type}: #{map.get($--variables, $type)};
```

Comment:
I think we should name this variable more explicitly for instance: `$--variable-map` or something, or maybe that's just redundant given we have `map.get` here. WDTY?

Reply:
Which is better, `$--variable-map` or `$--variables-map`?

> Because it is an internal variable, it has no effect on the user.

Reply:
checked the definitions of some other ui libraries and the examples given by sass. They usually don't use map as a suffix.
So I think `$--variables` is better.

Reply:
`$--variables` `s` means that it is a combination of many variables.